compose-examples/prysm-only/docker-compose.yaml

```diff
@@ -91,11 +91,27 @@ services:
     user: "1000"
     depends_on:
       - prometheus
+      - renderer
     ports:
       - 127.0.0.1:3000:3000
     volumes:
       - ./config/grafana/provisioning-p:/etc/grafana/provisioning
       - ./data/grafana:/var/lib/grafana
+    environment:
+      GF_RENDERING_SERVER_URL: http://renderer:8082/render
+      GF_RENDERING_CALLBACK_URL: http://grafana:3000
+      GF_LOG_FILTERS: rendering:debug
```

Comment:
debug necessary?

Reply:
not really, I used it for log msg for testing purpose, now it's removed